push.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
could you remove this and add to .gitignore

Reply:
Yes, sorry!!!